docs/running-on-yarn.md

```diff
@@ -441,6 +441,16 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
   <td>1.6.0</td>
 </tr>
+<tr>
+  <td><code>spark.yarn.am.clientModeExitOnErro</code></td>
+  <td>false</td>
+  <td>
+  In yarn-client mode when this is true, if driver got application report with final status of KILLED or FAILED,
```

Comment:
add , after mode and before when
use:
"if the driver received an application report"

Reply:
Done